desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -1,21 +1,32 @@
 [Desktop Entry]
 Type=Application
 Name[ar]=رَفل
+Name[cs]=Ruffle
+Name[es]=Ruffle
 Name[fr]=Ruffle
+Name[ja]=Ruffle
 Name[ko]=Ruffle
 Name[nl]=Ruffle
 Name[pl]=Ruffle
+Name[ru]=Ruffle
 Name[zh_CN]=Ruffle
 Name=Ruffle
 GenericName[ar]=مشغل الفلاش
+GenericName[cs]=Flash přehrávač
+GenericName[es]=Reproductor de Flash
 GenericName[fr]=Lecteur Flash
+GenericName[ja]=Flash Player
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash speler
 GenericName[pl]=Odtwarzacz Flash
+GenericName[ru]=Flash Player
 GenericName[zh_CN]=Flash 播放器
 GenericName=Flash Player
 Comment[ar]=شغِّل أفلام وألعاب الفلاش
+Comment[cs]=Hrajte flash hry & filmy
```

Comment:
I'm not sure whether Czech uses `&` in native sentences. It's possible that Crowdin suggested that `&` is present in the original translation and should also be present here, and the translator complied. I suspect `a` might be a better fit instead of `&`.

Reply:
The second question is whether we can use `hrajte` in relation to movies (`filmy`). I'm not sure about the usage in Czech, but in Polish there are separate verbs for playing games (`graj w gry`, similar to `hrajte hry`), and playing movies (`odtwarzaj filmy`, similar to `??? filmy`). Also `hrajte` seems to be plural here, but perhaps it shouldn't be?

Reply:
And I think `flash` should start with an uppercase letter.

Reply:
Added a comment on Crowdin